### ISSUE TITLE
apt-get update failing due to removal of Wheezy and Jessie (except LTS) from mirrors 

### ISSUE THREAD

**Issue by jimymodi** [confluentinc/cp-docker-images#714]
When I am doing `apt-get update` on base of `confluentinc/cp-kafka-rest:4.0.0`. It is failing with `Failed to fetch http://deb.debian.org/debian/dists/jessie-updates/main/binary-amd64/Packages  404  Not Found`
It is because [Removal of Wheezy and Jessie (except LTS) from mirrors](https://lists.debian.org/debian-devel-announce/2019/03/msg00006.html)

My Dockerfile for ref
```
FROM confluentinc/cp-kafka-rest:4.0.0

ENV TZ Asia/Kolkata

RUN apt-get update

RUN apt-get install -y uuid-runtime

COPY start-kafka-rest /usr/bin/start-kafka-rest

RUN chmod 755 /usr/bin/start-kafka-rest

ENTRYPOINT ["/usr/bin/start-kafka-rest"]
```

**Comment by mprimeaux** [comment]
I can confirm I also experience this for "confluentinc/cp-zookeeper:latest".

**Comment by slepkin** [comment]
This is also a problem for `confluentinc/cp-kafka-connect:latest`.

**Comment by andybryant** [comment]
Work arounds discussed here work... https://unix.stackexchange.com/questions/508724/failed-to-fetch-jessie-backports-repository
.. but really the base image should be updated from jessie since it's now considered an obsolete release: https://www.debian.org/releases/
The latest KSQL image doesn't have the same issue since it's based on stretch

**Comment by andybryant** [comment]
confluentinc/cp-enterprise-control-center:5.1.2 has same issue.

**Comment by ewencp** [comment]
Hey folks, we've merged this update to address the issue: https://github.com/confluentinc/cp-docker-images/pull/717 We will be moving the base image too stretch, but please keep in mind that there are compatibility issues with updating these, so this is the sort of change that can only be made across major release versions. Just changing base image in bugfixes would break some downstream consumers.

For existing releases, subsequent bugfix releases of the images for that major.minor version will include the linked PR.

**Comment by pdkproitf** [comment]
This is also a problem for `confluentinc/cp-kafka-connect:5.1.2`

**Comment by jinbowang** [comment]
Do we have plan to retrofit this config into older versions?

**Comment by maknihamdi** [comment]
All debian images have this error. I already test this workarround https://unix.stackexchange.com/questions/508724/failed-to-fetch-jessie-backports-repository/508840#508840 with other image, and now I applied it with kafka-connect. Works fine!

you have to add this line to your Dockerfile

```
RUN echo "deb [check-valid-until=no] http://cdn-fastly.deb.debian.org/debian jessie main" > /etc/apt/sources.list.d/jessie.list
RUN echo "deb [check-valid-until=no] http://archive.debian.org/debian jessie-backports main" > /etc/apt/sources.list.d/jessie-backports.list
RUN sed -i '/deb http:\/\/deb.debian.org\/debian jessie-updates main/d' /etc/apt/sources.list 

RUN apt-get -o Acquire::Check-Valid-Until=false update && ....

```

**Comment by ewencp** [comment]
This has already been fixed back to 3.2.x branch in https://github.com/confluentinc/cp-docker-images/pull/717, so all builds against updated bug fix release versions back to then would have this change. Tagged releases are already tagged, so we can't retroactively update the tag, but you can just cherry-pick the change against a branch started from the tag if you want to rebuild those versions yourself.
